plugins/eos/gs-plugin-eos.c

```diff
@@ -453,6 +462,20 @@ fake_os_upgrade_progress (GsPlugin *plugin)
 	return G_SOURCE_CONTINUE;
 }
 
+static gboolean
+updater_is_stalled (GsPlugin *plugin)
+{
+	GsPluginData *priv = gs_plugin_get_data (plugin);
+	GsApp *app = priv->os_upgrade;
+
+	/* in case the OS upgrade has been disabled */
+	if (priv->updater_proxy == NULL)
+		return EOS_UPDATER_STATE_NONE;
```

Comment:
`EOS_UPDATER_STATE_NONE` is not a `gboolean`. Should this be `return FALSE`?

Reply:
Argh. This was a c&p mistake. Thanks.